scripts/coverage.sh

```diff
@@ -12,6 +12,14 @@ else
   OUTPUT_DATENLORD=coverage/datenlord_cov.lcovrc
 fi
 
+# install and start etcd
```

Comment:
It's weird to start etcd service in the coverage script. I get the point that we need to get test result with the service on, but it's also not a practice to make `cargo test` depend on a third-party service.

Usually we use mock service in the test to make it clean.

Reply:
Ok. I'll change relate test to mock.